content/applications/general/email_communication/email_servers.rst

```diff
@@ -5,122 +5,122 @@ Send and Receive Emails in Odoo with an Email Server
-Office 365 SMTP relays are not compatible with Odoo Online unless you configure
-Odoo to :ref:`force the outgoing "From" address <email_communication/default_from>` .
 
-Please refer to `Microsoft's documentation
-<https://support.office.com/en-us/article/How-to-set-up-a-multifunction-device-or-application-to-send-email-using-Office-365-69f58e99-c550-4274-ad18-c805d654b4c4>`_
-to configure a SMTP relay for your Odoo's IP address.
+You can use an Office 365 server if you run Odoo on-premise. Office 365 SMTP relays are not
+compatible with Odoo Online unless you configure Odoo to :ref:`force the outgoing "From" address
+<email_communication/default_from>`.
+
+Please refer to `Microsoft's documentation <https://support.office.com/en-us/article/How-to-set-up-
+a-multifunction-device-or-application-to-send-email-using-Office-365-69f58e99-c550-4274-ad18-
+c805d654b4c4>`_ to configure a SMTP relay for your Odoo's IP address.
 
 How to use a G Suite server
 ---------------------------
-You can use an G Suite server for any Odoo hosting type.
-To do so you need to setup the SMTP relay service. The configuration steps are explained in 
-`Google documentation <https://support.google.com/a/answer/2956491?hl=en>`__.
+
+You can use an G Suite server for any Odoo hosting type. To do so you need to setup the SMTP relay
+service. The configuration steps are explained in `Google documentation <https://support.google.com
+/a/answer/2956491?hl=en>`__.
 
 Restriction
 -----------
-Please note that port 25 is blocked for security reasons on our SaaS and Odoo.sh platform. Try using
-465, 587, or 2525.
+
+Please note that port 25 is blocked for security reasons on our SaaS and Odoo.sh platform. Try
+using 465, 587, or 2525.
 
 .. _email_communication/default_from:
 
 Use a default "From" email address
 ----------------------------------
 
-Sometimes, an email's "From" (outgoing) address can belong to a 
-different domain, and that can be a problem.
+Sometimes, an email's "From" (outgoing) address can belong to a different domain, and that can be a
+problem.
 
-For example, if a customer with address *mary@customer.example.com* responds to
-a message, Odoo will try to redistribute that same email to other subscribers
-in the thread. But if the domain *customer.example.com* forbids that kind of
-usage for security, the Odoo's redistributed email would get
-rejected by some recipients' mail servers.
+For example, if a customer with address *mary\@customer.example.com* responds to a message, Odoo
+will try to redistribute that same email to other subscribers in the thread. But if the domain
+*customer.example.com* forbids that kind of usage for security, the Odoo's redistributed email
+would get rejected by some recipients' mail servers.
 
-To avoid those kind of problems, you should make sure all emails use a "From"
-address from your authorized domain.
+To avoid those kind of problems, you should make sure all emails use a "From" address from your
+authorized domain.
```

Comment:
Replace the entire sentence `To avoid those...authorized domain.` with:

`To avoid that problem, make sure all emails use a "From" address from the authorized domain.`